__tests__/app.test.js

```diff
@@ -93,3 +93,27 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200: should respond with an array of articles sorted by date", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        expect(articles).toHaveLength(13);
```

Comment:
I think this behaviour (sorted by) should ideally have been driven by its own separate test